src/components/Checkbox/Checkbox.jsx

```diff
@@ -107,6 +113,14 @@ export const Checkbox = forwardRef(
 Checkbox.propTypes = {
   id: PropTypes.string,
   className: PropTypes.string,
+  /**
+   * ClassName for icon container
```

Comment:
```suggestion
   * ClassName for checkbox sketch container
```
It's not must to be this description suggestion, but in my opinion the issue here is that developers can be confused because they are not understand how we implement our fake checkbox and if it's an icon or not, so I think the description need to be focused about that we affecting the fake checkbox and not about the fake checkbox structure.